rust/impl/willbe/commands/mod.rs

```diff
@@ -13,4 +13,51 @@ wtools::mod_interface!
   prelude mod each;
   
   protected use super::init::protected::*;
+
+  prelude use State;
+}
+
+mod private
+{
+  use core::ops::{ Deref, DerefMut };
+  use std::collections::HashMap;
+
+  #[ derive( Debug ) ]
+  /// Commands context
+  pub struct State( HashMap< String, wca::Command > );
+
+  impl State
+  {
+    /// Construct state object
+    pub fn new( val : HashMap< String, wca::Command > ) -> Self
+    {
+      Self( val )
+    }
+  }
+
+  impl Deref for State
+  {
+    type Target = HashMap< String, wca::Command >;
+
+    fn deref( &self ) -> &Self::Target
+    {
+      &self.0
+    }
+  }
+
+  impl DerefMut for State
+  {
+    fn deref_mut( &mut self ) -> &mut Self::Target
+    {
+      &mut self.0
+    }
+  }
+
+  impl Into< wca::Context > for State
```

Comment:
Why do you implement `Into` but not `From`? Does the implementation of `From` not implement `Into` for this case?